.travis/build_package.sh

```diff
@@ -125,66 +103,27 @@ cd $ROOT
     fi
   fi
   IFS=$' '
-  EXAMPLES="$ARG/examples/$ARG"
-  TEST="$ARG/test/$ARG" 
-  DEMOS=$ROOT/$ARG/demo/*
-
-  if [ -d "$ROOT/$EXAMPLES" ]
-  then
-    cd $ROOT/$EXAMPLES
-    if [ -f ./CMakeLists.txt ]; then
-      build_examples
-    else
-      for dir in ./*
-      do
-        if [ -f $dir/CMakeLists.txt ]; then
-          cd $ROOT/$EXAMPLES/$dir
-          build_examples
-        fi
-      done
-    fi
-  elif [ "$ARG" != Polyhedron_demo ]; then
-    echo "No example found for $ARG"
+  mkdir build
+  cd build
+  mkdir -p build-travis
+  cd build-travis
+  WITHDEMOS=ON
+  if [ "$ARG" = "Polyhedron" ]; then
+    WITHDEMOS=OFF
   fi
+  EXTRA_CXX_FLAGS=
+  case "$CC" in
+    clang*)
+      EXTRA_CXX_FLAGS="-Werror=inconsistent-missing-override"
+      ;;
+  esac
+  
+  
+  mytime cmake -DCMAKE_CXX_FLAGS="${CXX_FLAGS} ${EXTRA_CXX_FLAGS}" -DCGAL_DONT_OVERRIDE_CMAKE_FLAGS:BOOL=ON -DBUILD_TESTING=ON -DWITH_tests=ON -DWITH_examples=ON -DWITH_demos=$WITHDEMOS ..
+  mytime ctest -j2 -L $ARG --output-on-failure
```

Comment:
Warning! BIG WARNING!

- Before Travis was only testing the compilation of examples/tests/demos, and now it will test also the execution.
- `ctest -j2 -L Triangulation` will also trigger the tests for `Triangulation_2` and `Triangulation_3`: you have to refine the regex passed to the `-L` option.

Reply:
Ah yes sorry, you're right